tests/test_move_embargo_args.py

```diff
@@ -171,15 +279,72 @@ def test_after_now_01(self):
             self.temp_to_path,
             move=move,
             log=self.log,
+            # datasettype=["raw", "raw"],
+            # collections=["LATISS/raw/all", "LATISS/raw/all"],
+            datasettype=["raw"],
+            collections=["LATISS/raw/all"],
+            desturiprefix=self.temp_dest_ingest,
         )
 
+    # test the other datatypes:
+    # first goodseeingdeepcoadd
+
+    def test_raw_datatypes_should_fail(self):
+        """
+        Test that move_embargo_args runs for a list
+        of input datatypes
+        """
+        move = "False"
+        # now_time_embargo = "now"
+        # embargo_hours =  80.0 # hours
+        now_time_embargo = "2020-01-17 16:55:11.322700"
+        embargo_hours = 0.1  # hours
+        # IDs that should be moved to temp_to:
+        # lol 2019111300059 should be in the ids_moved
+        # list but I'm removing it to make sure the assertions
+        # fail
+        ids_moved = [
+            2019111300061,
+            2020011700002,
+            2020011700003,
+        ]
+        # IDs that should stay in the temp_from:
+        ids_remain = [
+            2020011700004,
+            2020011700005,
+            2020011700006,
+        ]
+
+        try:
+            is_it_there(
+                embargo_hours,
+                now_time_embargo,
+                ids_remain,
+                ids_moved,
+                self.temp_from_path,
+                self.temp_to_path,
+                move=move,
+                log=self.log,
+                datasettype=["raw"],
+                collections=["LATISS/raw/all"],
+                desturiprefix=self.temp_dest_ingest,
+            )
+        except AssertionError:
```

Comment:
You can handle this more simply via `@pytest.mark.xfail`.

Reply:
Done, and changed the testing to pytest in order to retrieve the xfails.